content/docs/server_setup/server_software.mdx

```diff
@@ -0,0 +1,222 @@
+---
+title: Server Software
+description: A detailed guide on which server software is right for you
+---
+
+import { Card, Cards } from "fumadocs-ui/components/card";
+
+# Introduction
+
+Everybody has different requirements when setting up their server. This guide will help you decide which server software is right for you.
+Each server software has its own uses and features, so it's important to choose the right one for your needs.
+
+<Callout type="info" title="Note">
+  Server Software(s) are commonly referred to as "Server Jar(s)". This is
+  because server softwares are usually distributed as a `.jar` file.
+</Callout>
+
+# Backend Servers
+
+Backend servers are the most common type of server software.
+There are an incomprehensible amount of backend servers available, however this section will cover the most recommended, stable and widely-used softwares.
+
+<Callout type="info" title="Note">
+  A "fork" is a copy of a software project that is modified in some way.\
+  They typically share the same codebase but have modified (or additional) features and optimizations.\
+  An example of "forks" include: Bukkit -> CraftBukkit -> Spigot -> Paper -> Pufferfish -> etc.
```

Comment:
Bukkit -> CraftBukkit

Bukkit is and always has been the API. CraftBukkit was the implementation. This is incorrect.

Reply:
Resolved in [b941455](https://github.com/admincraft-community/admincraftwiki/pull/1/commits/b941455642ba78b509aa65176e61c907ba41e212) hopefully